pep-0683.rst

```diff
@@ -503,11 +503,11 @@ to do its best effort of deallocating these instances normally.  Most
 of the module deallocation will now be handled by
 ``pylifecycle.c:finalize_modules()`` which cleans up the remaining
 modules as best as we can.  It will change which modules are available
-during __del__ but that's already defined as undefined behavior by the
-docs.  Optionally, we could do some topological disorder to guarantee
+during ``__del__``, but that's already defined as undefined behavior by the
```

Comment:
```suggestion
during ``__del__``, but that's already declared to be undefined behavior by the
```

Avoid confusing and repetitive "defined to be undefined" phrasing.

Reply:
Not sure about this one, will leave to @ericsnowcurrently to decide.

Reply:
Would you suggest a different word than "declared"? There are certainly other options ("documented", "stated", etc), but the point is "defined as undefined" is inherently oxymoronic, and there's no strong compelling need to use the very specific diction "defined" rather than one of its synonyms here that would justify this.

Reply:
To me "defined as undefined" sounds odd but not wrong, since "undefined behavior" is a well-defined (sorry) concept in programming language design. I don't think it matters much, so I'll just let Eric make the call when he reviews this PR.

Reply:
Sure, but as a copyeditor, I don't see why it is necessary for it to sound "odd" (and potentially confusing) when there are many equally- or more-appropriate synonyms for the first "defined" (unlike the second usage, which is part of the well-defined term "undefined behavior").

Here are a few other alternatives of the top of my head for @ericsnowcurrently to consider, if he prefers them, that would also avoid the problem:

```suggestion
during ``__del__``, but that's already stated to be undefined behavior by the
```

```suggestion
during ``__del__``, but that's already described as undefined behavior by the
```

```suggestion
during ``__del__``, but that's already characterized as undefined behavior by the
```

Reply:
I'd rather the following:

```suggestion
during ``__del__``, but that's already explicitly undefined behavior in the
```